src/main/java/org/gridsuite/sensitivityanalysis/server/service/SensitivityAnalysisService.java

```diff
@@ -94,6 +109,42 @@ public List<String> getProviders() {
                 .collect(Collectors.toList());
     }
 
+    public Long getFactorsCount(SensitivityFactorsIdsByGroup factorIds, UUID networkUuid, String variantId, Boolean isInjectionsSet) {
+        Long containersAttributesCount = 1L;
+        if (Boolean.TRUE.equals(isInjectionsSet)) {
+            containersAttributesCount *= factorIds.getIds().get(INJECTIONS).size();
+            factorIds.getIds().remove(INJECTIONS);
+        }
+        containersAttributesCount = getFactorsCount(factorIds, networkUuid, variantId, containersAttributesCount);
```

Comment:
```suggestion
        containersAttributesCount *= getFactorsCount(factorIds, networkUuid, variantId);
```
I think it would be cleaner to not mutate containersAttributesCount in the sub getFactorsCount method